lib/init.js

```diff
@@ -151,7 +151,12 @@ class Init extends EventEmitter {
     ]
     const result = await inquirer.prompt(questions)
     if (result.answer === 'y') {
-      this.dpObj = await Dataset.load(this.path_)
+      try {
+        this.dpObj = await Dataset.load(this.path_)
+      } catch (err){
+        err.message += '\nExisting descriptor file is invalid, please fix or delete it manually.'
```

Comment:
This always assumes that Error happened due to an invalid descriptor. May be we should first check if error message contains `Unexpected token` substring and only then replace `message` with our custom one. Wdyt? @AcckiyGerman

Reply:
Yes, you're right, it could be other types of errors, e.g. 'access' errors.
FIXED.